src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/configs/DeciderConfig.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.configs;
+
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.framework.core.RcaConf;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class DeciderConfig {
+
+    public static final String CACHE_BOUNDS_CONFIG_NAME = "cache-bounds";
+    public static final String CACHE_CONFIG_NAME = "cache-type";
+    public static final String WORKLOAD_CONFIG_NAME = "workload-type";
+    public static final String PRIORITY_ORDER_CONFIG_NAME = "priority-order";
+    public static final String FIELD_DATA_CACHE_UPPER_BOUND = "field-data-cache-upper-bound";
+    public static final String SHARD_REQUEST_CACHE_UPPER_BOUND = "shard-request-cache-upper-bound";
+    private static final double DEFAULT_FIELD_DATA_CACHE_UPPER_BOUND = 0.4;
+    private static final double DEFAULT_SHARD_REQUEST_CACHE_UPPER_BOUND = 0.05;
+    // Defaults based on prioritising Stability over performance.
+    private static final List<String> DEFAULT_WORKLOAD_PRIORITY = Arrays.asList("ingest", "search");
+    private static final List<String> DEFAULT_CACHE_PRIORITY = Arrays.asList("read-heavy", "write-heavy");
+
+    public Double fieldDataCacheUpperBound;
```

Comment:
It seems like these are written and then never read. Are we doing this on purpose? Will this only be configurable in the future?

Reply:
Nvm, I see what's happening here. Can you follow the Java standard practice and make these private and add getters for them?

Reply:
Have added for the ones which are not being used currently as well.